### PR TITLE
errors,version(ticdc): refine ErrCheckClusterVersionFromPD error

### LINKED ISSUE #4341: [CDC:ErrCheckClusterVersionFromPD]failed to request PD%!(EXTRA string=response status: 503 Service Unavailable)
### What did you do?

- 4 pd, 2 tidb, 7 tikv, 2 cdc
- test infra case: 360 pd_leader_switch_changefeed_admin_sync
- when pd leader switch one by one,meanwhile changefeed pause/resume

### What did you expect to see?

_No response_

### What did you see instead?

- cli chnangefeed resume failed:

2022-01-14T19:16:51.039Z	INFO	host/ticdc.go:220	Execute command on ticdc	{"command": "/cdc  cli  changefeed  resume \"--pd=http://upstream-pd.cdc-qihoo-360-testbed-tps-600887-1-433:2379\" \"--changefeed-id=qihoo360-mysql-task\"", "timeout": "10s", "ticdc peer": 0}
[2022/01/14 19:16:51.050 +00:00] [INFO] [pd.go:32] ["pd leader"] [leader-name=upstream-pd-2]
[2022/01/14 19:16:51.553 +00:00] [INFO] [pd.go:46] ["transfer pd leader success"] [leader-name=upstream-pd-1]
[2022/01/14 19:16:52.215 +00:00] [ERROR] [ticdc.go:61] ["run changefeed operation failed"] [output="{stdout: Usage:\n  cdc cli changefeed resume [flags]\n\nFlags:\n  -c, --changefeed-id string   Replication task (changefeed) ID\n  -h, --help                   help for resume\n      --no-confirm             Don't ask user whether to ignore ineligible table\n\nGlobal Flags:\n      --ca string          CA certificate path for TLS connection\n      --cert string        Certificate path for TLS connection\n  -i, --interact           Run cdc cli with readline\n      --key string         Private key path for TLS connection\n      --log-level string   log level (etc: debug|info|warn|error) (default \"warn\")\n      --pd string          PD address, use ',' to separate multiple PDs (default \"http://127.0.0.1:2379\"), stderr: Error: [CDC:ErrCheckClusterVersionFromPD]failed to request PD%!(EXTRA string=response status: 503 Service Unavailable)\n[CDC:ErrCheckClusterVersionFromPD]failed to request PD%!(EXTRA string=response status: 503 Service Unavailable), ExitCode: 1"] [stack="github.com/pingcap/test-infra/caselib/pkg/host.(*TiCDCHost).ChangeFeed\n\t/Users/lixia/source-code/test-infra/caselib/pkg/host/ticdc.go:61\ngithub.com/pingcap/test-infra/caselib/pkg/steps.(*changeFeedTask).Execute\n\t/Users/lixia/source-code/test-infra/caselib/pkg/steps/changefeed.go:49\ngithub.com/pingcap/test-infra/caselib/pkg/steps.(*loopTask).Execute\n\t/Users/lixia/source-code/test-infra/caselib/pkg/steps/task.go:95\ngithub.com/pingcap/test-infra/caselib/pkg/steps.(*Parallel).Execute.func1\n\t/Users/lixia/source-code/test-infra/caselib/pkg/steps/step.go:69\ngolang.org/x/sync/errgroup.(*Group).Go.func1\n\t/Users/lixia/go/pkg/mod/golang.org/x/sync@v0.0.0-20210220032951-036812b2e83c/errgroup/errgroup.go:57"]

### Versions of the cluster

Upstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):

```console
5.4.0
```

TiCDC version (execute `cdc version`):

```console
5.4.0
```

### PR DESCRIPTION
**Pull request by overvenus**
### What problem does this PR solve?

Issue Number: close #4341

### What is changed and how it works?

* Add actual error to error message
* Add retry hint.

### Release note 

```release-note
None
```

### CODE REVIEW

File: errors.toml [hunk 1]
```diff
@@ -143,7 +143,7 @@ changefeed update error: %s
 
 ["CDC:ErrCheckClusterVersionFromPD"]
 error = '''
-failed to request PD
+failed to request PD %s, please retry again later
```
Comment (overvenus) [inline errors.toml:146]:
```suggestion
failed to request PD %s, please try again later
```

File: pkg/errors/errors.go [hunk 2]
```diff
@@ -117,7 +117,7 @@ var (
 
 	// utilities related errors
 	ErrToTLSConfigFailed         = errors.Normalize("generate tls config failed", errors.RFCCodeText("CDC:ErrToTLSConfigFailed"))
-	ErrCheckClusterVersionFromPD = errors.Normalize("failed to request PD", errors.RFCCodeText("CDC:ErrCheckClusterVersionFromPD"))
+	ErrCheckClusterVersionFromPD = errors.Normalize("failed to request PD %s, please retry again later", errors.RFCCodeText("CDC:ErrCheckClusterVersionFromPD"))
```
Comment (overvenus) [inline pkg/errors/errors.go:120]:
```suggestion
	ErrCheckClusterVersionFromPD = errors.Normalize("failed to request PD %s, please try again later", errors.RFCCodeText("CDC:ErrCheckClusterVersionFromPD"))
```

**Review by 3AceShowHand — APPROVED** [review]

**Review by hi-rustin — APPROVED** [review]
It would be a good to do a manual test to see exactly what the error has become now.

Change LGTM.

**Review by leoppro — COMMENTED** [review]
LGTM

**Pull request merged**